resources/public/api/conf/1.0/checkpoints_restore.yaml

```diff
@@ -0,0 +1,59 @@
+post:
+  $ref: "./checkpoints_restore.yaml"
+  summary: Restore to checkpoint
+  description: |
+    This endpoint allows a developer to reset the stateful data back to the checkpoint created in the sandbox environment.
+  
+    A valid Checkpoint ID must be provided.
+  
+    When the developer restores the checkpoint the modified date of the data will be updated to the restored date.
+
+    **Note**: To prevent data loss or unexpected behaviour, the checkpoint restore should not be performed while other stateful checkpoint operations are in progress for the same National insurance number.
+
+  security:
+    - User-Restricted:
+        - write:self-assessment
+
+  parameters:
+    - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+    - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
+    - $ref: './common/pathParameters.yaml#/components/parameters/checkpointId'
```

Comment:
Could we give the checkpointId a "required" tag? 

i.e. something like the "required" label that the nino variable in the 'Create a Checkpoint' docs has.